src/addin/client-interfaces/addin-client-open-help-args.ts

```diff
@@ -1,5 +1,6 @@
 /**
  * Interface for launching the help tab from an add-in.
+ * @deprecated See _Deprecated help support_ in README
```

Comment:
Let's not deprecate the entire `AddinClientOpenHelpArgs` just yet.

Reply:
Instead, let's just deprecate the `helpKey` since it's what directly relates to the deprecated help widget,